tests/import.py

```diff
@@ -13,6 +13,29 @@ def test():
     # sanity check
     print("RPM conf dir:", rpm.expandMacro("%getconfdir"))
 
+    import site
+    import sys
+    from pathlib import Path
+
+    assert len(rpm.__path__) == 1
+    rpm_path = rpm.__path__[0]
+    search_paths = sys.path + site.getsitepackages()
+
+    for site_dir in search_paths:
+        try:
+            # is_relative_to is only present in python 3.9+
+            # relative_to raises a ValueError if the paths are not relative
+            # => retry
+            # if no exception is raised => found a match and break out
+            Path(rpm_path).relative_to(Path(site_dir))
+            break
+        except ValueError:
+            pass
+    else:
+        raise RuntimeError(
+            f"{rpm_path} is not relative to any python search path: {search_paths}"
+        )
+
```

Comment:
What's the point of this check? It breaks the ubuntu test case.

Reply:
I wanted to verify with this, that the imported rpm module belongs to the same interpreter. But the whole test is moot, if it is expected that python 3.11 uses the python 3.9 rpm module on CentOS/RHEL 9. I'll have to rework the test to correctly blow up on SLES and pass on RHEL

Reply:
Turns out that just trying to instantiate the `rpm.spec` module/class was enough for a regression test